e2e/runner/zevm.go

```diff
@@ -322,9 +322,10 @@ func (r *E2ERunner) WaitForBlocks(n int64) {
 	call := func() error {
 		return retry.Retry(r.waitForBlock(height.Height + n))
 	}
-
-	bo := backoff.NewConstantBackOff(time.Second * 5)
-	boWithMaxRetries := backoff.WithMaxRetries(bo, 10)
+	retryBuffer := uint64(20)
+	bo := backoff.NewConstantBackOff(time.Second * 6)
+	// #nosec G115 always in range
+	boWithMaxRetries := backoff.WithMaxRetries(bo, uint64(n)+retryBuffer)
```

Comment:
Q: is this change made because `WaitForBlocks` fails to work when `n` is too big?

Reply:
yeah , the retry loigc needs to wait for atleast n*6 secs